randovania/games/super_metroid/json_data/Wrecked Ship.txt

```diff
@@ -8,24 +8,28 @@ Extra - asset_id: 51
   > Door to Gravity Suit Room
       All of the following:
           Morph Ball and After Power Wrecked Ship
-          Grapple Beam or Space Jump
+          Any of the following:
+              Grapple Beam or Space Jump
+              # HBJ
+              Horizontal Infinite Bomb Jump (Expert) and Can Lay Bombs
+              # TODO: Damage Boost numbers. Does HJ affect this?
+              Damage Boosting (Beginner) and Normal Damage ≥ 300
+              # TODO: short charge from previous room?
+              Speed Booster and Shinespark (Advanced) and Disabled Door Lock Randomizer and Disabled Entrance Randomizer
```

Comment:
The shinespark would also require health, but I'm not sure if its possible.

Reply:
With shinespark: https://youtu.be/3SEmfFHrcfY
4 tap short charge (expert) + gravity
4 tap short charge (hypermode) without gravity (wait on water to fall and perfect movement)

Takes 91 health.

Reply:
For dboosting, 2 hits is perfect movement. HJ makes it harder.
Expert: 60*2 = 120 Damage
Advanced 60*3 = 180 Damage
Intermediate 60*4 = 240 Damage

Reply:
damage numbers dont need to be laid out like this, because there's an "adjust damage numbers" setting, which by default is 1.5
i set this to advanced but with 120 damage for now.